mne/io/ctf/tests/test_ctf.py

```diff
@@ -0,0 +1,167 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os
+from os import path as op
+import shutil
+
+import numpy as np
+from nose.tools import assert_raises, assert_true
+from numpy.testing import assert_allclose, assert_array_equal, assert_equal
+
+from mne import pick_types
+from mne.transforms import apply_trans
+from mne.io import Raw, read_raw_ctf
+from mne.utils import _TempDir, run_tests_if_main, slow_test
+from mne.datasets import testing
+
+ctf_dir = op.join(testing.data_path(download=False), 'CTF')
+ctf_fname_continuous = 'testdata_ctf.ds'
+ctf_fname_1_trial = 'testdata_ctf_short.ds'
+ctf_fname_2_trials = 'testdata_ctf_pseudocontinuous.ds'
+ctf_fname_discont = 'testdata_ctf_short_discontinuous.ds'
+ctf_fname_somato = 'somMDYO-18av.ds'
+ctf_fname_catch = 'catch-alp-good-f.ds'
```

Comment:
155 MB total:

7 for catch
1 for som
20 for testdata_ctf
36 for testdata_ctf_pseudocontinuous
10 for testdata_short
7 for testdata_short_discontinuous

x2 for all the MNE-C converted versions.